README.md

```diff
@@ -1,2 +1,3 @@
 # University
 Study project about university 
+More details will be available soon.
```

Comment:
Add a domains and fields to the Readme file.

Reply:
Added